### ISSUE TITLE
Cross project testing

### ISSUE THREAD

**Issue by rhornung67** [LLNL/RAJA#1605]
Set up GitLab CI so that RAJAPerf tests run with each PR merge to RAJA develop.

**Comment by long58** [comment]
This is a pretty simple fix, can I just add it to my existing PR?

**Comment by adrienbernede** [comment]
@long58 please see my comment here: https://github.com/LLNL/RAJA/pull/1644#discussion_r1642359362

From the issue description, it looks like you need to run RAJAPerf each time there is a new commit in develop, so you should make sure the trigger is active on that branch.

If there is a misunderstanding, let’s meet and discuss it ;).

**Comment by long58** [comment]
Yeah, I guess that's true... When I first read it, my initial thought was to run the RAJAPerf tests on the branch that was going to be merged into develop, but it did not occur to me that github merge requests and gitlab were separate... Which is indeed an issue. Sounds like the original solution that was implemented would probably work just fine for this use case, as it always runs the RAJAPerf tests on the develop branch.

**Comment by rhornung67** [comment]
Closing. Will be part of performance data monitoring work.
